web_src/less/_base.less

```diff
@@ -1390,6 +1422,23 @@ i.icon.centerlock {
   color: var(--color-text-light);
 }
 
+.ui.active.label {
+  background: var(--color-secondary);
+  border-color: var(--color-secondary);
+  color: var(--color-text-dark);
+}
+
+.ui.active.label:hover,
+a.ui.active.label:hover {
+  background: var(--color-secondary-dark-1);
+  color: var(--color-text-dark);
+}
+
+.ui.form .field.error .ui.multiple.selection.dropdown > .red.label {
+  color: #e7b6b6 !important;
+  background: #871717 !important;
```

Comment:
Should use some vars here.

Reply:
I invented these colors as I said in OP. I meant exactly this by "formalizing" - they have to fit into some scheme. I'd rather not use extra long names for this single use instance. I just didn't spot on cases where these colors could be shared... yet. Any suggestions?

Reply:
Maybe `--color-error-text` and `--color-error-bg`. My point is that we want all colors the live in vars at `:root` so they can be easily altered for theming. You should define suitable colors for arc-green as well, the background would be too bright for that theme.

Reply:
That would mean we'd better go deeper and override more stuff from Fomantic UI, which was not the main goal of this PR. Of course, since we are on it, I'll see what I can do. I mean to put all colors used in Semantic Dropdown into `_base.less` where I could basically just switch colors for the highlighted label (the invented colors are generated with a color picker by modifying the HSL values - I just happened to have come up with something so close to switching around the foreground and background colors - you could suggest better ones).